src/Login/Login.test.js

```diff
@@ -1,3 +1,53 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
 import Login from './Login';
+import { BrowserRouter } from 'react-router-dom';
+
+describe('Login', () => {
+  it('should render a login form', () => {
+    const fakeAddUser = jest.fn();
+    render (<BrowserRouter><Login addUser={fakeAddUser}/></BrowserRouter>);
+    expect(screen.getByText('Login Here')).toBeInTheDocument();
+    expect(screen.getByPlaceholderText('Enter your email')).toBeInTheDocument();
+    expect(screen.getByPlaceholderText('Enter your password')).toBeInTheDocument();
+    expect(screen.getByText('login')).toBeInTheDocument();
+  })
+
+  it('should display an alert if the user enters incorrect login info', () => {
```

Comment:
I imagine, based on the review, that this is the type of test that we don't need